cmd/root.go

```diff
@@ -272,6 +272,10 @@ func startServer(opts *common.Options) error {
 		walletrpc.RegisterDarksideStreamerServer(server, service)
 	}
 
+	if !opts.Darkside {
+		common.StartPriceFetcher(dbPath, chainName)
```

Comment:
Hmm, now that you mention it, I may have added that conditional in haste, incorrectly. I was thinking well _of course_ we don't want lightwalletd to go out and ping these websites in dark mode, since it's supposed to be self-contained (no `zcashd`) testing only. But maybe it's okay to do that.

OR, we could stub out the two price gRPCs and just return hardcoded fake data. That way internet access wouldn't be needed to do (full) darkside testing. Which do you think would be better?

Reply:
we should return stub data so integration tests can be reliable and deterministic